Formula/mariadb.rb

```diff
@@ -151,6 +151,9 @@ def caveats; <<-EOS.undent
 
     To connect:
         mysql -uroot
+
+    Using brew services will place a plist file at #{ENV["HOME"]}/Library/LaunchAgents/#{plist_name}
```

Comment:
If you don't like the 3rd line about enabling verbose mode for brew services it could be ditched. The first 2 lines give somebody a fighting chance of knowing what to try and look for to change the bind address and that it's even a thing that is being overridden.